### DISCUSSION TITLE
Ribbon cable detach sound with pack powered off

### DISCUSSION THREAD

**Discussion by Toy203** [gpstar81/GPStar-proton-pack#309] [Ideas]
Since we now have an history of sounds played while pack powered off (ciclotron opening, power on effect) it would be cool to have a proper sound for Ribbon Cable detach/reattach even with the pack is powered off. Guess it'd be something more "mechanical" than "electrical". 

If i'm not mistaken you can clearly hear the "attach" effect on FE scene with Phoebe in the kitchen.

**Comment by nomakewan** [comment]
This has been added as of commit [27dabfb](https://github.com/gpstar81/GPStar-proton-pack/commit/27dabfb9cf2474454de2e113595a8a94e1aefd76) and will be included in the next firmware update. Thank you for your suggestion!
